src/pot_process/pot_process.cpp

```diff
@@ -0,0 +1,222 @@
+/// Small utility to check powersoftau output and to compute the evaluation of
+/// Lagrange polynomials at tau.
+
+#include "snarks/groth16/powersoftau_utils.hpp"
+
+#include <boost/program_options.hpp>
+#include <fstream>
+
+using namespace libzeth;
+namespace po = boost::program_options;
+using ppT = libff::default_ec_pp;
+
+// -----------------------------------------------------------------------------
+// cli_options
+// -----------------------------------------------------------------------------
+
+// Usage:
+//     pot-process [<options>] <powersoftau file> <degree>
+//
+// Options:
+//     -h,--help              This message
+//     -v,--verbose           Verbose
+//     --check                Check pot well-formedness and exit
+//     --out <file>           Write the lagrange polynomial values to this file
+//                            ("lagrange-radix2-<n>")
+//     --lagrange-degree <l>  Use degree l instead of n (l < n)
+//     --dummy                Create dummy powersoftau data (for testing only!)
+class cli_options
+{
+public:
+    po::options_description desc;
+    po::options_description all_desc;
+    po::positional_options_description pos;
+
+    std::string argv0;
```

Comment:
Maybe use `command`? I saw that you renamed the `argv0` attribute of the other subcommands by `subcommand`, so it might be a good idea to follow a similar approach here.

Reply:
OK.  Renamed to `command`.